### PR TITLE
Add a news post for playtest-20170405

### PR DESCRIPTION
**Pull request by abcdefg30**
page-fullpage

It might be a bit tight to get this out today. Not too bad if we don't though, I can just increase all "version" numbers then.

### CODE REVIEW

File: content/news/playtest-20170405.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
+You probably have already guessed the bad news, the buyable hats from [our April Fools post](/news/april-20170401/) are not going to be in the next OpenRA release. Don't be too sad however, as this ought to be the final playtest before the next release! So be sure to [grab yourself an installer](/download/) and use this last chance to report bugs and give us feedback.
+
+As we're trying to make our release (candidates) as stable as possible, this playtest doesn't include many big changes compared to the last playtest. However, the Tiberian Dawn map pool got an extensive overhaul, with over 50 maps being added! Other minor touch-ups, like the changes to the Red Alert map pool or the last minute balance fine tweaks are listed in the [complete changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog).
```
Comment (abcdefg30) [inline content/news/playtest-20170405.md]: Note to self: The changelog link probably needs an update later on.

**Comment by abcdefg30** [comment]
Add ~~bad writing~~ a battlefield news entry.

File: content/news/playtest-20170405.md [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+
+<div style="text-align:center" markdown="1">
+![new td map pool](/images/news/20170405-td-map-pool.png)
+
+The new map pool offers map for both young and old Tiberian Dawn players.
```
Comment (obrakmann) [inline content/news/playtest-20170405.md]: *maps

File: content/news/playtest-20170405.md [hunk 3]
```diff
@@ -0,0 +1,28 @@
...
+
+![yak and gapgen](/images/news/20170405-yak-gap.png)
+
+Yaks and MiG got a slight sight range increment versus gap generation.
```
Comment (obrakmann) [inline content/news/playtest-20170405.md]: *MiGs

File: content/news/playtest-20170405.md [hunk 4]
```diff
@@ -0,0 +1,28 @@
...
+</div>
+
+As always, if you encounter a bug, or have any other feedback, feel free to open a ticket on our [issue tracker](bugs.openra.net), contact us [on IRC](http://webchat.freenode.net/?channels=openra), create a topic on our [community forum](http://www.sleipnirstuff.com/forum/viewforum.php?f=80), or leave a message in the comments below.
```
Comment (obrakmann) [inline content/news/playtest-20170405.md]: The issue tracker link probably won't work without the `http://` in front.

File: content/news/playtest-20170405.md [hunk 5]
```diff
@@ -0,0 +1,28 @@
...
+
+<hr>
+
+<div style="text-align:center" markdown="1">
+![ragl season III logo](/images/news/20170405-ragl-III.png)
+</div>
+
+On a different note, we just entered the first break round of the [Red Alert Global League](http://www.sleipnirstuff.com/forum/viewtopic.php?f=82&t=20000), with 129 games being played so far!
+If you want to get an impression of how RA:GL matches look, check FiveAces' newest cast out:
```
Comment (obrakmann) [inline content/news/playtest-20170405.md]: *check out FiveAces' latest cast:

File: content/news/playtest-20170405.md (hunk 4 above)
Comment (abcdefg30) [inline content/news/playtest-20170405.md]: It worked for me, but I can add that (won't hurt ^^).

**Comment by abcdefg30** [comment]
Updated.

File: content/news/playtest-20170405.md [hunk 6]
```diff
@@ -0,0 +1,28 @@
...
+As we're trying to make our release (candidates) as stable as possible, this playtest doesn't include many big changes compared to the last playtest. However, the Tiberian Dawn map pool got an extensive overhaul, with over 50 maps being added! Other minor touch-ups, like the changes to the Red Alert map pool or the last minute balance fine tweaks are listed in the [complete changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog).
+
+<div style="text-align:center" markdown="1">
+![new td map pool](/images/news/20170405-td-map-pool.png)
+
+The new map pool offers maps for both young and old Tiberian Dawn players.
+
+![yak and gapgen](/images/news/20170405-yak-gap.png)
+
+Yaks and MiGs got a slight sight range increment versus gap generation.
```
Comment (pchote) [inline content/news/playtest-20170405.md]: Language nit: got &rarr; received

**Comment by obrakmann** [comment]
Updated

**Pull request merged**